b2share/modules/schemas/errors.py

```diff
@@ -25,38 +25,45 @@
 
 from __future__ import absolute_import
 
+from doschema.errors import JSONSchemaCompatibilityError
+from invenio_rest.errors import FieldError, RESTException
 
-class B2ShareSchemasError(Exception):
+
+class B2ShareSchemasError(RESTException):
     """Base class for the B2Share Schemas module."""
-    pass
 
+    def __init__(self, description=None, code=None):
+        """Constructor"""
+        super(B2ShareSchemasError, self).__init__()
+        self.code = code if code is not None else 500
```

Comment:
This sets the default `code` value to 500 in all the subclasses' instances, which I believe is not the intended behaviour. Do you mean:

````
class B2ShareSchemasError(RESTException):
    code = 500
    def __init__(self, description=None, code=None):
        ....
        if code:
            self.code = code
````